<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Triangle</class>
 <widget class="QMainWindow" name="Triangle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Triangle</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="1">
     <widget class="PersistentLineEdit" name="lineEditb"/>
    </item>
    <item row="3" column="1">
     <widget class="PersistentLineEdit" name="lineEditB"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>c</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="PersistentLineEdit" name="lineEdita"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>a</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="PersistentLineEdit" name="lineEditc"/>
    </item>
    <item row="5" column="1">
     <widget class="PersistentLineEdit" name="lineEditC"/>
    </item>
    <item row="0" column="1">
     <widget class="PersistentLineEdit" name="lineEditA"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>b</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PersistentLineEdit</class>
   <extends>QLineEdit</extends>
   <header>persistentlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditA</tabstop>
  <tabstop>lineEditB</tabstop>
  <tabstop>lineEditC</tabstop>
  <tabstop>lineEdita</tabstop>
  <tabstop>lineEditb</tabstop>
  <tabstop>lineEditc</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
